README.md

```diff
@@ -58,6 +60,41 @@ Pug transforms the above to:
 </html>
 ```
 
+## API
+
+```ts
+import {
+  compile,
+  Options,
+  render,
+  renderFile,
+} from "https://raw.githubusercontent.com/lumeland/pug/master/mod.ts";
```

Comment:
In lume I'm using jsdelivr https://github.com/lumeland/lume/blob/master/deps/pug.ts#L1 so I can use tags and seems more easy to use.

Reply:
Did you need to create a new release? If I try to visit https://cdn.jsdelivr.net/gh/lumeland/pug/mod.ts the file was not found.

Reply:
I'll create a new tag tomorrow